packages/ui-patterns/Dialogs/ConfirmationModal.tsx

```diff
@@ -1,80 +1,216 @@
-import { MouseEventHandler, PropsWithChildren, useEffect, useState } from 'react'
-import { Button, Modal } from 'ui'
+// import { MouseEventHandler, PropsWithChildren, useEffect, useState } from 'react'
```

Comment:
should we just remove all these? am okay with just leaving it if its temp

Reply:
fixe